goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf

```diff
@@ -0,0 +1,41 @@
+data "aws_s3_bucket" "goobi_data" {
+  bucket = "${var.goobi_items_bucket_name}"
+}
+
+data "aws_iam_policy_document" "allow_s3_access" {
+  statement {
+    actions = [
+      "s3:GetObject",
+    ]
+
+    resources = [
+      "${data.aws_s3_bucket.goobi_data.arn}",
+      "${data.aws_s3_bucket.goobi_data.arn}/*",
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "read_from_goobi_items_q" {
```

Comment:
This is provided as an output of our SQS module – try `${module.goobi_items_queue.read_policy}`.